htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -373,6 +373,15 @@ UPDATE llx_propal SET date_signature = date_cloture WHERE date_signature IS NULL
 
 
 ALTER TABLE llx_product ADD COLUMN batch_mask VARCHAR(32) NULL;
+ALTER TABLE llx_product ADD COLUMN life_duration VARCHAR(6);
```

Comment:
Can you rename "life_duration" into "lifetime"
"control_periodicity" into "inspection_frequency"

Reply:
sure thing, I’m on it